x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/details_page/index.tsx

```diff
@@ -179,181 +267,55 @@ export const AgentConfigDetailsPage: React.FunctionComponent = () => {
               agentConfig={agentConfig}
             />
           ) : null}
-          <EuiTitle size="m">
-            <h3>
-              <FormattedMessage
-                id="xpack.ingestManager.configDetails.agentsSummaryTitle"
-                defaultMessage="Enrolled agents"
-              />
-            </h3>
-          </EuiTitle>
-          <EuiSpacer size="l" />
-          {agentStatusIsLoading || !agentStatus ? (
-            <Loading />
-          ) : agentStatusError ? (
-            <FormattedMessage
-              id="xpack.ingestManager.configDetails.agentStatusNotFoundErrorTitle"
-              defaultMessage="Unable to load enrolled agents status"
-            />
-          ) : (
-            <EuiFlexGroup gutterSize="xl">
-              <EuiFlexItem grow={false}>
-                <EuiTitle size="xs">
-                  <h5>
-                    <FormattedMessage
-                      id="xpack.ingestManager.configDetails.agentsTotalTitle"
-                      defaultMessage="Total"
-                    />
-                  </h5>
-                </EuiTitle>
-                <EuiFlexGroup alignItems="center">
-                  <EuiFlexItem grow={false}>
-                    <EuiTitle size="l">
-                      <span>{agentStatus.total}</span>
-                    </EuiTitle>
-                  </EuiFlexItem>
-                  <EuiFlexItem grow={false}>
-                    {/* TODO: Make this link to filtered agents list and change to real agent count */}
-                    <ConnectedLink color="primary" path={`/fleet/agents`}>
-                      <FormattedMessage
-                        id="xpack.ingestManager.configDetails.viewAgentsLinkText"
-                        defaultMessage="View agents"
-                      />
-                    </ConnectedLink>
-                  </EuiFlexItem>
-                </EuiFlexGroup>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiTitle size="xs">
-                  <h5>
-                    <FormattedMessage
-                      id="xpack.ingestManager.configDetails.eventsTitle"
-                      defaultMessage="Events"
-                    />
-                  </h5>
-                </EuiTitle>
-                <EuiFlexGroup alignItems="center">
-                  <EuiFlexItem grow={false}>
-                    <EuiTitle size="l">
-                      <span>{agentStatus.events}</span>
-                    </EuiTitle>
-                  </EuiFlexItem>
-                </EuiFlexGroup>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiTitle size="xs">
-                  <h5>
+
+          {tabId === '' && (
```

Comment:
What do you think about using nested route for tabs?  https://reacttraining.com/react-router/core/guides/philosophy/nested-routes

Reply:
Yeah, I'm good with that and will make this cleaner.